SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -130,6 +132,9 @@ namespace SpatialConstants
 	const Schema_FieldId SHUTDOWN_MULTI_PROCESS_REQUEST_ID					= 1;
 	const Schema_FieldId SHUTDOWN_ADDITIONAL_SERVERS_EVENT_ID				= 1;
 
+	const Schema_FieldId PING_EVENT_ID										= 1;
```

Comment:
Why there indentation change?

Reply:
Github shows tabs as 6 spaces even though they're 4 in code.

Reply:
I mean why the indentation change from the lines around it.

Reply:
Although it looks like github does 8

Reply:
Discussed offline - I'm reading the tabs wrong.